10_Getting_Started/20_Build/10_Linux.md

```diff
@@ -282,6 +282,19 @@ See our page presenting [video tutorial for compilation on Linux](../../video-tu
 
 
 
+# Run SOFA
+
+## with the SOFA GUI
+To run SOFA, locate and execute the application called `runSofa`. For more detailed information on how to use the application, you can refer to the [page dedicated to runsofa](../../../using-sofa/runsofa/). This documentation will provide you with further guidance on using SOFA effectively.
+
+
+## within a Python environment
+
+To use SOFA within a Python3 environment, the section "using Python3" details how to [set up your environment on various operating systems](https://sofapython3.readthedocs.io/en/latest/content/Installation.html#using-python3).
```

Comment:
I suggest to report the changes you made here to all the places you initially copy/paste.